### PR TITLE
Fix #10362: Datatable allow multiple SummaryRow

### LINKED ISSUE #10362: Datatable: Allow multiple SummaryRows
@Rapster we just got this requirement in from a client. We are replacing an old desktop PowerBuilder application (yes PowerBuilder) and it has double SummaryRow's like in this screenshot.

image

When we tried to do 2 summaryRow we realized it only allowed for 1 so we couldn't render this result the way we wanted.  I am submitting a PR to allow for multiple SummaryRow.

### PR DESCRIPTION
**Pull request by melloware**
Fix #10362: Datatable allow multiple SummaryRow

### CODE REVIEW

**Review by Rapster — APPROVED** [review]

**Comment by Rapster** [comment]
should it be 13.0.1 or 14.0.0 as it's no defect?

**Comment by melloware** [comment]
I think its up to us. I need this for a client so normally i would ask to be patched through their PRO account to 13.0.1 but i think all of that is changing?

**Comment by Rapster** [comment]
All i know is that, if we follow sem version, this fix should go to 13.1.0

**Comment by melloware** [comment]
yep you are right let me move to 14.0.0 for now until we can discuss further on Discord?

**Comment by tandraschko** [comment]
yep, i dont think we will have something like 13.1.0

File: primefaces/src/main/java/org/primefaces/component/datatable/DataTable.java [hunk 1]
```diff
@@ -664,15 +664,16 @@ public void setSelectAll(boolean selectAll) {
         getStateHelper().put(InternalPropertyKeys.selectAll, selectAll);
     }
 
-    public SummaryRow getSummaryRow() {
+    public List<SummaryRow> getSummaryRow() {
```
Comment (tandraschko) [inline primefaces/src/main/java/org/primefaces/component/datatable/DataTable.java]: getSummaryRows

**Comment by Rapster** [comment]
Should we go back to 2 digits then? PTek did the switch in 10 and always turned out to be useless,

**Comment by tandraschko** [comment]
actually yes

**Pull request merged**